index.html

```diff
@@ -5610,21 +5613,31 @@ <h3>Language and Base Direction</h3>
 
         <pre class="example nohighlight" title="Design pattern for natural language strings">
 "<a>property</a>": {
-  "value": "<span class="highlight">The string value</span>",
-  "lang": "<code>LANGUAGE</code>"
-  "dir": "<code>DIRECTION</code>"
+  "@value": "<span class="highlight">The string value</span>",
+  "@language": "<code>LANGUAGE</code>"
+  "@direction": "<code>DIRECTION</code>"
 }
         </pre>
 
+        <p>
+When the language value object is used in place of a string value, the object
+MUST contain a `@value` property whose value is a string, and SHOULD contain a
+`@language` property whose value is a string containing a well-formed
+`Language-Tag` as defined by [[BCP47]], and MAY contain a `@direction` property
+whose value is a text direction string defined by the `@direction` attribute in
```

Comment:
Consider linking "text direction" to the definition in the I18N glossary.

Reply:
@aphillips there's no discrete "text direction" definition in the I18N glossary. Would you recommend I reference [base direction](https://www.w3.org/TR/i18n-glossary/#dfn-base-direction) or [paragraph direction](https://www.w3.org/TR/i18n-glossary/#dfn-paragraph-direction) instead?

Reply:
@BigBlueHat Use base direction here.